### PR TITLE
test: cryptoapis integration tests

### PR DESCRIPTION
**Pull request by austinborn**
## Description

Cryptoapis integration tests

## Changes

- One integration test for each of: `balance`, `bc_info`, and `crypto` endpoints
- Add `crypto` as supported name for endpoint

## Steps to Test

`yarn && yarn setup && yarn test cryptoapis/test/integration`

## Quality Assurance

- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [x] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
- [x] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)

### CODE REVIEW

**Review by austinborn — COMMENTED** [review]
Note to self: update package version and CHANGELOG

**Comment by austinborn** [comment]
> Note to self: update package version and CHANGELOG

Done

File: .changeset/hot-onions-impress.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@chainlink/proof-of-reserves-adapter': minor
+'@chainlink/cryptoapis-adapter': minor
+---
+
+Add cryptoapis integration tests
```
Comment (boxhock) [inline .changeset/hot-onions-impress.md:6]: Integration tests isn't part of the source files that are changed, the real change here is that the way exports are done has been changed.

I also don't think we need to include `@chainlink/proof-of-reserves-adapter` explicitly here. @justinkaseman what do you think?

Comment (austinborn) [reply .changeset/hot-onions-impress.md:6]: Cryptoapis src export change

Comment (austinborn) [reply .changeset/hot-onions-impress.md:6]:
```suggestion
Cryptoapis src export change
```

File: .changeset/hot-onions-impress.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+'@chainlink/proof-of-reserves-adapter': minor
+'@chainlink/cryptoapis-adapter': minor
```
Comment (boxhock) [inline .changeset/hot-onions-impress.md:3]:
```suggestion
'@chainlink/cryptoapis-adapter': minor
```

**Review by boxhock — DISMISSED** [review]
Wanna give this PR a quick rebase so we can merge?

File: .changeset/hot-onions-impress.md (hunk 2 above)
Comment (austinborn) [reply .changeset/hot-onions-impress.md:3]: Just to be sure, you want to remove `proof-of-reserves-adapter` even with the change in `packages/composites/proof-of-reserves/src/balance.ts`?

Comment (boxhock) [reply .changeset/hot-onions-impress.md:3]: Oooh, sorry didn't see that. This probably don't need to change then, thanks!

**Review by boxhock — APPROVED** [review]

**Pull request merged**